lib/game/widget_builder.dart

```diff
@@ -0,0 +1,202 @@
+import 'package:flutter/widgets.dart';
+import '../gestures.dart';
+import 'game.dart';
+
+class WidgetBuilder {
```

Comment:
We may have some confusion with this name since `WidgetBuilder` is a very common typedef for callbacks inside the framework.

Reply:
I just moved this class from another file, that name has been on Flame for sometime now, I do agree with you though, it can cause confusion, but we should handle a rename of this, on other PR.

Reply:
Agreed